### ISSUE TITLE
Invitation of user to Digital Identity

### ISSUE THREAD

**Issue by veitmo** [amosproj/amos2022ss04-digital-identity#48]
## User story
1. As a {HR-employee}
2. I want / need { to send an invitation to the user }
3. So that {he can connect to his newly created digital identity in the Lissi wallet}

## Acceptance criteria
* Process:
  * HR-employee enters Name, Surname, email-address in frontend and issues the creation of a new DI
  * backend creates new DI and receives URL #25 
  * backend generates QR-code from URL #47 
  * backend sends email to corresponding email-address with QR-code #9 
  * result: QR-code is successfully scanned by Lissi Wallet
* described process must run

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release

**Comment by Idontker** [comment]
Done in sprint 5 https://github.com/amosproj/amos2022ss04-digital-identity/commit/8b9cdf974b0592bb76a4d87a3d35fe9834c38272
